test/fixtures/apiGatewayAsync/serverless.yml

```diff
@@ -0,0 +1,17 @@
+service: service
```

Comment:
For such small use cases, we don't usually create new fixtures, but rather go with reusing and adjusting an existing one with `configExt` arg to `runServerless` and it would be the best way to go in this case. You can e.g. reuse `apiGateway` fixture or just plain `function` fixture